### PR TITLE
Fix refund calculator conversion

### PR DESCRIPTION
**Pull request by Tomanhez**
Before:
Screenshot 2019-09-25 at 12 04 53

### CODE REVIEW

File: tests/Behat/Context/Ui/RefundingContext.php [hunk 1]
```diff
@@ -76,6 +76,18 @@ public function decideToRefundProductWithVeryLongComment(
         $this->orderRefundsPage->refund();
     }
 
+    /**
+     * @When /^I decide to refund (\d+) "([^"]*)" products with "([^"]*)" payment$/
+     */
+    public function iDecideToRefundProductsWithPayment(int $amout, string $productName, string $paymentMethod): void
+    {
+        for($number = 0;$number<$amout;$number++) {
```
Comment (lchrusciel) [inline tests/Behat/Context/Ui/RefundingContext.php]: Give them some space :) 

```suggestion
        for($number = 0; $number < $amout; $number++) {
```

File: tests/Application/src/Migrations/Version20190607135638.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20190607135638 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE TABLE sylius_avatar_image (id INT AUTO_INCREMENT NOT NULL, owner_id INT NOT NULL, type VARCHAR(255) DEFAULT NULL, path VARCHAR(255) NOT NULL, UNIQUE INDEX UNIQ_1068A3A97E3C61F9 (owner_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET UTF8 COLLATE UTF8_unicode_ci ENGINE = InnoDB');
```
Comment (lchrusciel) [inline tests/Application/src/Migrations/Version20190607135638.php]: Shouldn't be part of this PR, I suppose. Is this file need to make this PR work?

File: tests/Behat/Context/Ui/RefundingContext.php [hunk 3]
```diff
@@ -76,6 +76,18 @@ public function decideToRefundProductWithVeryLongComment(
         $this->orderRefundsPage->refund();
     }
 
+    /**
+     * @When /^I decide to refund (\d+) "([^"]*)" products with "([^"]*)" payment$/
+     */
+    public function iDecideToRefundProductsWithPayment(int $amout, string $productName, string $paymentMethod): void
+    {
+        for($number = 0; $number < $amout; $number++) {
```
Comment (GSadee) [inline tests/Behat/Context/Ui/RefundingContext.php]:
```suggestion
        for ($number = 0; $number < $amount; $number++) {
```

File: tests/Behat/Context/Ui/RefundingContext.php [hunk 4]
```diff
@@ -76,6 +76,18 @@ public function decideToRefundProductWithVeryLongComment(
         $this->orderRefundsPage->refund();
     }
 
+    /**
+     * @When /^I decide to refund (\d+) "([^"]*)" products with "([^"]*)" payment$/
+     */
+    public function iDecideToRefundProductsWithPayment(int $amout, string $productName, string $paymentMethod): void
```
Comment (GSadee) [inline tests/Behat/Context/Ui/RefundingContext.php]:
```suggestion
    public function iDecideToRefundProductsWithPayment(int $amount, string $productName, string $paymentMethodName): void
```

**Comment by pamil** [comment]
Thanks, Tomasz! :tada:
